CSS/mob-style.scss

```diff
@@ -0,0 +1,295 @@
+@use 'variables' as *;
+
+.navbar {
+  padding: 1rem 1.5rem;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+
+  &__link {
+    color: $clr-primary-700;
+    animation-name: rotate-logo;
+    animation-duration: 1.5s;
+    animation-iteration-count: 1;
+  }
+
+  &__hamb,
+  &__close {
+    cursor: pointer;
+    z-index: 100;
+  }
+}
+
+.head::before {
+  content: '';
+  position: absolute;
+  background-color: $clr-neutral-100;
+  background-image: url('https://raw.githubusercontent.com/ArthurGC/portfolio-microverse/add-set-up-mobile-version--portfolio/images/shapes.png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-origin: content-box;
+  border-bottom-left-radius: 5rem;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.head {
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+  padding: 6rem 1.5rem;
+  z-index: 10;
+
+  &__title {
+    color: $clr-neutral-800;
+    width: 10ch;
+
+    span {
+      display: block;
+      width: 10ch;
+    }
+  }
+
+  &__dsc {
+    margin: 0.8rem 0;
+    color: $clr-neutral-600;
+  }
+
+  &__link {
+    text-transform: uppercase;
+  }
+
+  &__icons {
+    margin: 0.5rem 0;
+    align-items: center;
+    justify-content: space-between;
+    width: 11rem;
+  }
+}
+
+.works {
+  &__cards {
+    grid-template-rows: repeat(3, auto);
+    row-gap: 5.5rem;
+    justify-content: center;
+    padding: 7.125rem 1.5rem;
+  }
+
+  &__card {
```

Comment:
- [ ] There is a section missing in your design, this is the partners section in the index page. Kindly add it.

Your design
![image](https://user-images.githubusercontent.com/81932478/190175055-b82d1fe4-b037-4130-b0ef-96be1984da47.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/190175159-23f4c113-362f-4b23-a152-500eddbcec80.png)

- Note to use flexbox to align the items in this section.